### ISSUE TITLE
Copy+paste errors in postgres docs

### ISSUE THREAD

**Issue by OrangeDog** [saltstack/salt#51213]
In particluar, the `maintainance_db` parameter is documented incorrectly for most states (having been copied from `postgres_language`), but the whole set probably needs a look.

https://github.com/saltstack/salt/blob/b3c1be2/salt/states/postgres_privileges.py#L143

**Comment by dwoz** [comment]
@OrangeDog Thanks for reporting this. Are you able to make a PR with the corrections?

**Comment by OrangeDog** [comment]
No, because I don't know what the correct documentation should be.
